Source/ZoomNet/Resources/Phone.cs

```diff
@@ -110,7 +110,7 @@ public Task UpdateCallHandlingSettingsAsync(
 			};
 
 			return _client
-				.PatchAsync($"/phone/extension/{extensionId}/call_handling/settings/{settingType.ToEnumString()}")
+				.PatchAsync($"/v2/phone/extension/{extensionId}/call_handling/settings/{settingType.ToEnumString()}")
```

Comment:
The base URI in the ZoomNetClient already contains `v2` therefore you don't need it here. The issue is that you have a `/`  prefix. Simply remove the extra `/` and it should solve the problem. Like this:
```suggestion
				.PatchAsync($"phone/extension/{extensionId}/call_handling/settings/{settingType.ToEnumString()}")
```

Reply:
Yeah, thanks, fixed!